lib/rules/no-await-sync-events.ts

```diff
@@ -41,18 +56,22 @@ export default createTestingLibraryRule<Options, MessageIds>({
 				type: 'object',
 				properties: {
 					eventModules: {
-						type: 'array',
-						minItems: 1,
-						items: {
-							enum: VALID_EVENT_MODULES,
-						},
+						default: FIRE_EVENT_OPTION,
+						oneOf: [
+							{ type: 'string', enum: VALID_EVENT_MODULES },
```

Comment:
I would keep this as an array only tbh as the name is representing multiple values

Reply:
Since the counterpart rule is singular, I would rename that to multiple in the next major release and only support passing an array (even if it's only 1 value) in both rules tbh

Reply:
Makes sense, I'll do that.